### ISSUE TITLE
Accessibility: Signal Message button should state an explicit action

### ISSUE THREAD

**Issue by alexstine** [signalapp/Signal-Android#13314]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

Currently, the button to send a message is labeled Signal Message. It would be more clear if the button stated a specific action such as Send Signal Message.

### Steps to reproduce
- Enable TalkBack: https://support.google.com/accessibility/android/answer/6007100?sjid=5755393204757638446-NC
- Open Signal.
- Select a chat.
- Select the compose field and type a message.
- Notice how the button text for sending the message is Signal Message.

**Actual result:** The button is labeled Signal Message.
**Expected result:** For better accessibility, the button should state exactly the action it takes. Text should read Send Signal Message.

### Device info

**Device:** Google Pixel 7
**Android version:** 14
**Signal version:** 6.40.5

**Comment by alexstine** [comment]
Still an issue.

**Comment by wtimme** [comment]
I just checked this in Signal Android 7.19.2, and the TalkBack reads "Send message" when selecting the button for sending the message.

I did a bit of digging and belive that this was resolved with e5652197eba9d29df7850888888d07df7ed154d8. Could you take another look, @alexstine?

**Comment by alexstine** [comment]
Resolved, thanks.
